File: features/featureOne/src/main/java/de/gfred/feature/one/di/FeatureOneBuilderModule.java
package de.gfred.feature.one.di;

import de.gfred.core.services.di.UserServiceModule;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;
import de.gfred.feature.one.FeatureOneActivity;

@Module
public abstract class FeatureOneBuilderModule
{
    @ContributesAndroidInjector(modules = {FeatureOneActivityModule.class, UserServiceModule.class})
    public abstract FeatureOneActivity contributeMainActivity();
}

File: features/featureOne/src/main/kotlin/de/gfred/feature/one/di/FeatureOneBuilderModule.kt
package de.gfred.feature.one.di

import dagger.Module
import dagger.android.ContributesAndroidInjector
import de.gfred.core.services.di.StringServiceModule
import de.gfred.feature.one.FeatureOneActivity

@Module
abstract class FeatureOneBuilderModule {
    @ContributesAndroidInjector(modules = [FeatureOneActivityModule::class, StringServiceModule::class])
    abstract fun contributeMainActivity(): FeatureOneActivity
}
